README.md

```diff
@@ -1,96 +1,40 @@
-[![Build Status](https://dev.azure.com/AzureDevOpsCliOrg/AzureDevOpsCli/_apis/build/status/Master%20Branch/Azure%20DevOps%20CLI%20-%20Merge%20GitHub?branchName=azuredevopscli-dev)](https://dev.azure.com/AzureDevOpsCliOrg/AzureDevOpsCli/_build/latest?definitionId=25?branchName=azuredevopscli-dev)
+# Azure DevOps Extension for Azure CLI
 
-# Azure Devops
+The Azure DevOps Extension for Azure CLI adds Pipelines, Boards, Repos, Artifacts and DevOps commands to the Azure CLI 2.0. 
 
-Azure Devops CLI is a new command line interface for [Azure Devops Service](https://azure.microsoft.com/en-in/services/devops/) 
 
-> Azure Devops CLI is currently in preview but we encourage you to give it a try and provide feedback (or contribute). 
+> Azure DevOps extension is currently in preview but we encourage you to give it a try and provide feedback (or contribute).
+With the release of Azure DevOps extension, we will be retiring the preview version of VSTS CLI. The Azure DevOps extension has functional parity with the VSTS CLI and we aim to depreciate the preview version of VSTS CLI by April, 2019. 
 
-## Get started
 
-Refer to the [install guide](https://aka.ms/azure-devops-cli-docs-install) for detailed install instructions for Windows, Mac, and Linux.
+# Installation
 
-Also refer to the ["get started" guide](https://aka.ms/azure-devops-cli-docs-getstarted) for in-depth instructions including first-time setup steps.
+1. [Install Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli). 
+You must have at least `v2.0.48`, which you can verify with `az --version` command.
```

Comment:
It was 2.0.49 as far as I remember. @gauravsaralMs Please confirm.

Reply:
yes, it is defined [here](https://github.com/Microsoft/vsts-cli/blob/azuredevopscli-dev/azure-devops/azext_devops/azext_metadata.json)

Reply:
Corrected.